README.md

```diff
@@ -201,7 +201,9 @@ To manually initiate a backup:
   a new EC2 instance:
   - Go the [Systems Manager Command History]
   - Find a recent successful backup
-  - Select the backup and click Rerun
+  - Open the command details and check the parameters to make sure it is for the
+    Jenkins instance you want to back up (jenkins or jenkins-prod)
+  - At the top of the command details page, click the Rerun button
```

Comment:
I don't know whether it's worth specifying where the bit that tells you whether it's jenkins or jenkins-prod is in the docker filter `dkr.ecr.eu-west-2.amazonaws.com/jenkins` It's not immediately obvious at first glance. I don't mind though, if you think it's clear then I'm happy.

Reply:
Yeah, it is a bit fiddly to find. I've added a bit more of an explanation - how's this version?

Reply:
Looks good, cheers